test/transpiler/transpiler_array_test.ts

```diff
@@ -196,5 +196,118 @@ describe('Transpiler', () => {
 
             expect(() => wrapper.setFunctionName('arrayExport').setOutParameters([71, 72]).call([71, 72])).to.throw();
         });
+
```

Comment:
I couldn't find any test that uses an array expression in a function call:
```javascript
testFunction([1, 2, 3]);
```

Or one that uses an array expression directly:
```javascript
[1, 2, 3][1] + 2 == 4;
```

Reply:
We decided to not allow this logic in our sublanguage.